File: node/update-into-elastic.js
const util = require('util');
const request = require("request-promise");

let esconn = require("../esconn.json");

// update indexed document
request.post({
    uri                    : esconn.uri + '/node/profile/278/_update?pretty',
    resolveWithFullResponse: true,
    json                   : true,
    body                   : {
        "doc": {
            "userphone": "lol"
        }
    }
}).then((response) => {
    console.log('update indexed document');
    console.log(`elasticsearch response : ${util.inspect(response.body, false, null)}`);
    return true
});

File: python/update-into-elastic.py
import json
import requests

esconn = json.load(open('../esconn.json'))


r = requests.post("%s%s" % (esconn['uri'],'/node/profile/278/_update?pretty'),
json={
        "doc": {
            "uri": "pitoune"
        }
    })
print(r.status_code, r.reason)
parsed = json.loads(r.content)
print (json.dumps(parsed, indent=4, sort_keys=True))







